src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/fragment/FeedFragment.java

```diff
@@ -24,66 +24,79 @@
     private static final String TAG = "FeedFragment";
 
     private FeedAdapter adapter;
-    private String feedName;
+    private final String feedName;
     private ListView mList;
     private List<FeedItem> mFeedItems;
 
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
```

Comment:
Whenever the feed fragment is created, we're always passing in a bundle with a name argument. There's also multiple checks before that to make sure we have a valid name. That said, I'm returning an empty list of feed items now whenever we encounter a null or empty string name. I also removed ``setFeedName`` (wasn't being used anyway).